src/components/DateInput/index.test.tsx

```diff
@@ -59,8 +60,7 @@ describe('DateInput component', () => {
 
   it('updates input value when user selects a day in the picker', () => {
     fireEvent.click(screen.getByRole('gridcell', { name: 'Tue Jul 07 2020' }));
-    expect(screen.getByRole('textbox').value).toBe('07/07/2020'); // MM/DD/YYYY === 10 chars
-    expect(screen.getByRole('textbox').value).toHaveLength(10); // MM/DD/YYYY === 10 chars
```

Comment:
Condensing both of these into one is a fantastic call. If the first test passes, the length is guaranteed. Very good pickup.